results.tex

```diff
@@ -28,16 +28,21 @@ \subsection{Underlying technology}
 Our goals include (1) to list regular Machine Learning algorithms that are commonly used in a distributed setting; (2) to find algorithms to determine the best parameters for the aforementioned algorithms; (3) to find a balance between computation time, communication overhead, and accuracy; and (4) to minimize the number of bits sent over the network, so that the system is not bottlenecked by network bandwidth.\\
 Designing a generic system that enables distribution of  regular Machine Learning efficiently is challenging, because every algorithm has its own communication patterns \cite{Jia14}\cite{Newman09}\cite{Rich13}\cite{Smola10}\cite{Takac13}\cite{Tsi12}.
 
-\subsubsection{Machine Learning algorithms}
-Machine Learning algorithms learn to make predictions or decisions based on data. Every Machine Learning algorithm has its own pros and cons. Machine Learning algorithms can be divided into categories based on some of their characteristics:
+\subsubsection{Machine Learning algorithms / parameter optimization}
+Machine Learning algorithms learn to make predictions or decisions based on data. Every Machine Learning algorithm has its own pros and cons. These can be divided into categories based on some of their characteristics:
```

Comment:
"These" isn't very elegant here, both because of the structure of the first sentence and the singular reference to "algorithm" in the next. Maybe "The algorithms"?